tools/tools.yml

```
@@ -248,7 +248,7 @@
       badges: pypi, travis, azure, coveralls, conda, site
 
 - name: Large-data rendering
-  intro: Tools for server-side rasterizing/aggregating data before visualization.
+  intro: Tools for rasterizing/aggregating data before visualization, to allow  in some cases server-side.
```

Sorry; somehow I must have submitted a version in progress, not the final one:

```suggestion
  intro: Tools for rasterizing/aggregating data before visualization, which can make rendering faster, allow larger datasets, or make it feasible to use remote datasets (with server-side rendering).
```